src/libs/UnreadIndicatorUpdater/updateUnread/index.website.js

```diff
@@ -10,6 +10,10 @@ import CONFIG from '../../../CONFIG';
  */
 function updateUnread(totalCount) {
     const hasUnread = totalCount !== 0;
+
+    // There is a Chrome browser bug that causes the title to revert back to the previous when we are navigating back. Setting the title to an empty string
+    // seems to improve this issue.
+    document.title = '';
```

Comment:
Haha weirddd. Do you think it's useful to link the chrome browser bug here so in the future if it gets fixed we can remove this hack?

Reply:
Yeah the thing is... it's not really well documented or at least I couldn't find a specific bug. 

All the chrome bugs related to `document.title` are here some of them are pretty old and still not fixed:

https://bugs.chromium.org/p/chromium/issues/list?q=document.title&can=2

Also came across some SO posts where people mention things not working on Chrome but working fine in other browsers:

https://stackoverflow.com/questions/65252275/chrome-not-updating-title
https://stackoverflow.com/questions/68369635/document-title-change-delay-in-chrome-edge

Normally I would say try to find a "better" solution - but think we can leave the original issue open and keep looking into it in this case.

Reply:
Makes sense to me, thanks for explaining!